R-package/tests/testthat/test_helpers.R

```diff
@@ -7,8 +7,8 @@ require(vcd, quietly = TRUE)
 
 float_tolerance = 5e-6
 
-# disable some tests for Win32
-win32_flag = .Platform$OS.type == "windows" && .Machine$sizeof.pointer != 8
+# disable some tests for 32-bit environment
+flag_32bit = .Machine$sizeof.pointer != 8
```

Comment:
What's wrong with our 32 bit build?

Reply:
Some of the unit tests will fail because 32-bit platform uses x87 unit, whereas 64-bit uses SSE. See #3723 for context. In the long run, we should re-design tests so that they work equally well on 32-bit and 64-bit systems. 

Support for 32-bit systems is mandated by CRAN submission policy.

Reply:
Tracking issue: #5578